lib/open_food_network/bulk_coop_report.rb

```diff
@@ -44,25 +46,12 @@ def header
     end
 
     def search
-      permissions.visible_orders.complete.not_state(:canceled).search(params[:q])
+      OpenFoodNetwork::Reports::LineItems.search_orders(permissions, params)
```

Comment:
No need to specify `OpenFoodNetwork` here. This code is already within that namespace.

Reply:
You are right, thanks. I changed it.